tutor/templates/apps/openedx/config/partials/auth.yml

```diff
@@ -1,7 +1,6 @@
 SECRET_KEY: "{{ OPENEDX_SECRET_KEY }}"
 AWS_ACCESS_KEY_ID: "{{ OPENEDX_AWS_ACCESS_KEY }}"
 AWS_SECRET_ACCESS_KEY: "{{ OPENEDX_AWS_SECRET_ACCESS_KEY }}"
-CONTENTSTORE: null
```

Comment:
For the record, this line is no longer necessary in cms/envs/production.py since this commit: https://github.com/openedx/edx-platform//commit/7d2a8340c5e997bf4ca43ca537a91e8f0962df1f#diff-aa2b311852d707265f0324225f3d9178bed527d71c84cd296de4de6a8d2c190dL388